src/plugins/codeStyle/index.ts

```diff
@@ -125,6 +128,18 @@ export default class CodeStyle {
                     diagnostics.push(messages.noPrint(s.tokens.print.range, noPrint));
                 }
             },
+            LiteralExpression: e => {
+                if (validateColorStyle && e.token.kind === TokenKind.StringLiteral) {
+                    validateColorStyle(e.token.text, e.token.range, diagnostics);
+                }
+            },
+            TemplateStringExpression: e => {
+                if (validateColorStyle) {
+                    for (const quasi of e.quasis.map(x => x.expressions).flat()) {
```

Comment:
IMHO I think the linter should bail if there is more than 1 quasi and if it has more than one expression - we shouldn't try to look for colours in a complex template literal, because that's not a use case for setting UI colours, isn't it?

Reply:
Yeah that's fair. I guess we're only validating whole strings, so we should do the same with template strings.

Reply:
I pushed those changes.